docs: update pipelines/add-pipeline article to add warning about `CUSTOM` operation (#857)
devopness/devopness: docs/pipelines/add-pipeline.md

1. On Devopness, navigate to a project then select an environment
1. Find the `Applications` card
1. Click `View` in the `Applications` card to see a list of existing `Applications`
1. In the list of applications find the application where you want to add a pipeline and click in the `NAME` of the application

> 1. On the left corner of the list click `PIPELINES`

Reviewer: ```suggestion
1. Click the `Pipelines` tab
```

1. On the right corner of the list click `MANAGE`
1. Provide a `name` for the new pipeline

> 1. Select the `OPERATION`

Reviewer: ```suggestion
1. Select an `Operation`
```

>     > Don't use the `CUSTOM` operation, it is not fully supported yet

Reviewer: ```suggestion
    > Don't use the `Custom` operation, it is not fully supported yet
```

> 1. Provide the `max parallel instances` for the pipeline

Reviewer: ```suggestion
1. Provide a value for the `Max parallel actions` field
```

Author: I copied your suggestions into my own commit, because when you made them, I had already made another commit correcting some of the spelling errors, and then the interface did not release the 'commit' of your suggestions.

1. In the list of applications find the application where you want to add a pipeline and click `DETAILS`

> 1. On the upper-right corner of the applications details view click `SETTINGS`

Reviewer: I believe this step also needs to be updated, as I couldn't find a `SETTINGS` UI element to click at.

1. Use the drop-down menu to choose `PIPELINES`
1. On the right corner of the list click `MANAGE`
1. Provide a name for the new pipeline
1. Select the `OPERATION`

>     > Don't use the `CUSTOM` operation, it is not fully supported

Reviewer: ```suggestion
    > Don't use the `CUSTOM` operation, it is not fully supported yet
```

> 1. Provide the limit for max parallel instances of this pipeline

Reviewer: `max parallel instances`?
I'd recommend mentioning the exact name of the field in the form on which the field is displayed.

<img width="846" alt="image" src="https://github.com/devopness/devopness/assets/1943318/8ef06262-919b-4ed7-8ecd-2256a735cc48">
